config.py

```diff
@@ -262,15 +262,15 @@ class Production(Live):
         },
         'send-daily-seller-email': {
             'task': 'app.tasks.mailchimp.send_new_briefs_email',
-            'schedule': crontab(hour=6, minute=0)
+            'schedule': crontab(hour=18, minute=0)
```

Comment:
Sorry Tobias, I think I gave you a bad steer on this one - I think this should be 17 i.e. 5PM.